python/tests/test_pyvw.py

```diff
@@ -192,4 +193,95 @@ def test_parse():
 
     ex = model.parse(["| a:1 b:0.5", "0:0.1:0.75 | a:0.5 b:1 c:2"])
     assert len(ex) == 2
+
+    # To test the prediction on multiline examples
+    finish = model.finish_example(ex)
+    assert model.predict(ex) == [0.0, 0.0]
     del model
+
+def test_namespace_id():
+    vw_ex = vw(quiet=True)
+    ex = vw_ex.example('1 |a two features |b more features here')
+    nm1 = pyvw.namespace_id(ex, 2)
```

Comment:
This example has 3 namespaces. Constant, a and b. If you are testing namespaces please assert all of them.

Reply:
Sure, I will test for all of them.